packages/vite/package.json

```diff
@@ -1,18 +1,34 @@
 {
   "name": "vite",
   "version": "3.0.0-alpha.1",
+  "type": "module",
   "license": "MIT",
   "author": "Evan You",
   "description": "Native-ESM powered web dev build tool",
   "bin": {
     "vite": "bin/vite.js"
   },
-  "main": "dist/node/index.js",
-  "types": "dist/node/index.d.ts",
+  "main": "./dist/node/index.js",
+  "module": "./dist/node/index.js",
+  "types": "./dist/node/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/node/index.d.ts",
+      "import": "./dist/node/index.js",
+      "require": "./index.cjs"
+    },
+    "./client": {
+      "types": "./client.d.ts"
+    },
+    "./terser": {
+      "require": "./dist/node-cjs/terser.cjs"
```

Comment:
TS complains here as missing declaration file.

Reply:
What version of TS and what “module” and/or “moduleResolution” settings are you using in your tsconfig.json file?

Reply:
Tried with TS 4.6 (node) and 4.7-rc1 (nodenext) with "module" being "esnext"

Reply:
If you mean `"./terser"`, it is being removed here https://github.com/vitejs/vite/pull/8049/files#diff-949e137b027a941e9a8dabb980971ab5b7147f895fc06ab2ff0bb0923d981606L23